bin/cleanup-test-resources

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+# -----------------------------------------------------------------------------
+# Cleanup orphaned test resources from failed CI runs
+#
+# This script finds and destroys resources from test runs that failed to
+# clean up after themselves. It targets resources with the project tag pattern
+# "plt-tst-act-*" that are older than a specified age.
+#
+# Usage:
+#   cleanup-test-resources [--dry-run] [--age-hours HOURS] [PROJECT_NAME]
+#
+# Arguments:
+#   PROJECT_NAME (optional) - Specific project to clean up (e.g., plt-tst-act-12345)
+#                             If not provided, finds all matching projects
+#
+# Options:
+#   --dry-run        - List resources that would be deleted without deleting them
```

Comment:
Doesn't have to be now, but could be useful to have a periodic GH Action that basically runs this and notifies if there's any resources, kinda like the `scan-orphaned-environments.yml` and `check-infra-deploy-status.yml` workflows do for project instances.